bitnami/redis/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: redis
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 21.1.6
+version: 21.1.8
```

I know there is another PR in the oven, but since the review process is isolated for each PR, we need to bump the version to the next number and then rebase the PR that is merged second from `main`.
```suggestion
version: 21.1.7
```